ex_webrtc/integration_test/test_videoroom/config/config.exs

```diff
@@ -33,16 +33,15 @@ config :logger, :console,
   format: "$time $metadata[$level] $message\n",
   metadata: [:request_id]
 
-config :logger,
-  compile_time_purge_matching: [
-    [level_lower_than: :info]
-  ]
```

Comment:
Why did you remove it?

Reply:
It's quite often that for tests I have to add debug logs. So that means both removing this entry (`compile_time_purge_matching`) and `logger: level`.

After this change there is a single place to change log level (`logger: level`)

Reply:
I would still leave it as it is because the engine and endpoints create a lot of debug logs, and this config removes them at compile time instead of ignoring invoked calls. It's about performance.

Reply:
It's only for tests, but I can revert this change.